test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Microsoft.ML.Benchmarks
 {
+    [WarmupCount(8)] // It helps to reduce the standard deviation of these tests.
```

Comment:
The normal user is unlikely to pre-train a model 8 times before training their model. This will be representative of the steady state reach when a model is retrained many times, but not very representative of the normal user's interaction w/ ML.net. 

Do we know what's causing the time difference between the first run and the later runs? The first run is most representative of what a normal user will experience.

Reply:
@adamsitnik can you give us a better view here ? Increasing the warmup iterations leads to reducing the standard deviation here

Reply:
@justinormont do u want me to reduce it ?

cc @danmosemsft

Reply:
> can you give us a better view here ?

@Anipik Unfortunately, it's not that simple. To find out why given benchmark behaves differently for different warmup counts we would have to profile it. It could be that OS gets warmed up and reading the input files becomes faster or anything like that.

> The normal user is unlikely to pre-train a model 8 times before training their model.

@justinormont I agree. In that case we should set the WarmupCount to 0, IterationCount to 1 and LaunchCount to 20. Which means that BenchmarkDotNet is going to start a new process 20 times and each time execute the benchmark only once, without any warmup (the real use case) and just exit the process.

Reply:
I will revert it to 1 for this PR to get merged, we can later follow up with 2 config files as adam suggested